src/util/user-home-dir.js

```diff
@@ -4,7 +4,8 @@ import ROOT_USER from './root-user.js';
 
 const path = require('path');
 
-const userHomeDir =
-  process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : require('os').homedir();
+export const home = require('os').homedir();
+
+const userHomeDir = process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : home;
```

Comment:
Ah actually @BYK  I see what you're saying. Yeah I don't think the platform check is even necessary given that line of code in root-user.js